### ISSUE TITLE
[feat] get delivery location on signup

### ISSUE THREAD

**Issue by opxdelwin** [tech-mySanji/customer_app_archive#1]
When the customer is registering in the app ,the device location is to be accessed and the pincode is requested as a mandatory field upon filling :-
 - If the device location is in the pincode, the map points to the device location and asks the customer to confirm the delivery location, following which he has to enter the complete address.
 - If the device location is not in the pincode entered, the map gives a zoomed out view of the pincode and the customer can drag and zoom to pin the precise location.

This needs to be updated by the delivery person's app in case of an incorrect address after verification.

**Comment by abinbabuabu** [comment]
Customer should be able to proceed without entering the full address details during signup. Basic details like full name and pincode. While ordering the if address is not added, add address option will be shown. ( Geo-location retrieved from device is part of the address object).

Nb: Users can also maintain multiple addresses.

**Comment by opxdelwin** [comment]
Right, on signup save exact location. On checkout enter exact address.
The multiple address feature has been implemented as of now? If not, we'll be saving it in the DB? This might lead to some legal complications if not taken care of properly.

**Comment by abinbabuabu** [comment]
Will discuss this, i have few more open points, will discuss with joe and confirm.

**Comment by abinbabuabu** [comment]
<img width="410" alt="image" src="https://github.com/tech-mySanji/customer_app_archive/assets/33547175/09f11f24-3fb4-49a0-9c3b-081c14ed42c6">

Also one more field with be there in this page. ie, phone number, we will always fill this field with user's login phone number. They can change it if they need.  Field name changes - Full Name to **Reciever's Name**. Title to **Delivery Address.**

**Comment by opxdelwin** [comment]
ok
